### PR TITLE
Add support for explicit render complete event

### LINKED ISSUE #30: Explicit rendering flag
Under consideration: Allow for a flag to notify when rendering is complete.

Need to find cases where the current loading function is insufficient. Do you have an example? Please comment below!

### PR DESCRIPTION
**Pull request by samuelli**
Closes #30.

### CODE REVIEW

File: test/app-test.js [hunk 1]
```diff
@@ -106,3 +106,10 @@ test('screenshot is an image', async(t) => {
   t.is(res.header['content-type'], 'image/jpeg');
   t.is(res.body.length, parseInt(res.header['content-length']));
 });
+
+test.only('explicit render event ends early', async(t) => {
```
Comment (aomarks) [inline test/app-test.js]: remove .only

File: README.md [hunk 2]
```diff
@@ -50,6 +50,14 @@ Available options:
 When setting query parameters as part of your URL, ensure they are encoded correctly. In JS,
 this would be `encodeURIComponent(myURLWithParams)`.
 
+### Explicit rendering event
+In some cases, like if you have a video playing on your page, it can be difficult to determine
+when your page has loaded. You can fire an event to indicate that the page has completed
+loading and rendering.
```
Comment (aomarks) [inline README.md]: Explain what happens if you don't fire the event?

File: test/app-test.js (hunk 1 above)
Comment (samuelli) [inline test/app-test.js]: Done.

File: README.md (hunk 2 above)
Comment (samuelli) [inline README.md]: Rewrote this whole thing to explain how the loading function works and how you can then choose to fire an event if you like.

**Pull request merged**